lib/facter/gluster.rb

```diff
@@ -78,11 +73,12 @@
           end
         end
       end
+      # rubocop:disable Metrics/BlockNesting
       if volume_options
         volume_options.each do |vol, opts|
           Facter.add("gluster_volume_#{vol}_options".to_sym) do
             setcode do
-              opts.join(',')
+              opts.join(';')
```

Comment:
+1 for splitting into 2 PRs if possible.  That will make it easier to get reviews and merged.  And *if* you're feeling really bored/generous, feel free to completely refactor the hideousness going on in volume.pp in a 3rd :)